vars/asfMavenTlpPlgnBuild.groovy

```diff
@@ -38,18 +38,21 @@ def call(Map params = [:]) {
 	// minimum, LTS, current and next ea
     def jdks = params.containsKey('jdks') ? params.jdks : params.containsKey('jdk') ? params.jdk : ['8','11','17']
     def jdkMin = jdks[0];
-    def mavens = params.containsKey('maven') ? params.maven : ['3.2.x','3.3.x','3.5.x','3.6.x']
+    def mavens = params.containsKey('maven') ? params.maven : ['3.6.x','3.8.x']
     // def failFast = params.containsKey('failFast') ? params.failFast : true
     // Just temporarily
     def failFast = false;
-    def siteJdks = params.containsKey('siteJdk') ? params.siteJdk : ['8','17']
-    def siteMvn = params.containsKey('siteMvn') ? params.siteMvn : '3.6.x'
+    def siteJdks = params.containsKey('siteJdk') ? params.siteJdk : ['8']
```

Comment:
should be at minimum 11, there are difference for javadoc

Reply:
uhm not sure as javadoc can fail with jdk 1.8 and not 11.  and the other way around :)
as we are 1.8 for most of the plugins etc maybe we need to ensure it works with this at least (as release folks might use 1.8 to build their release.